src/features/homeFeature/components/MangaListView.tsx

```diff
@@ -1,16 +1,32 @@
-import { MangaSummary } from "../../../models/topManga";
-import MangaCard from "./MangaCard";
-import topManga from "../../../mockData/topManga.json";
+import MangaCard from "../../../sharedComponents/MangaCard";
 import ListHeader from "./ListHeader";
+import { useLocation } from "react-router-dom";
+import { MangaDetails } from "../../../models/mangaDetails";
 
 const MangaListView = () => {
-  const mangaList = topManga["30"] as MangaSummary[];
+  const { state } = useLocation();
+
+  if (!state)
+    return (
+      <div className="flex flex-col items-center justify-center">
```

Comment:
A h1 might have been more appropriate here than a div (or h2/h3 whatever is the most fitting in this context.